root/etc/e-smith/events/actions/nethserver-nextcloud-conf

```diff
@@ -47,4 +47,5 @@ else
         exit 0
     fi
     OCC "db:add-missing-indices"
+    OCC "db:convert-filecache-bigint"
```

Comment:
Why this is needed?

Reply:
This command is to optimize some database column.
It has been showed from the admin web page as a warning.
It says that this operation is not automatically done because it can be computationally intensive.
Executing it more times, does not cause any problems.